solutions/bronze/usaco-1301.mdx

```diff
@@ -0,0 +1,83 @@
+---
+id: usaco-1301
+source: USACO Bronze 2022 February
+title: Watching Mooloo
+author: Chu Minjae
+---
+[Official Analysis (C++, Java, Python)](http://www.usaco.org/current/data/sol_prob3_bronze_feb23.html)
+
+## Explanation
+
+This question is solved through greedy algorithm. 
+
+To start with setting variables, as the input varies from 1 to 10^14, it is better to use ```long long``` rather than ```integer``` types. 
+
+Variable ```s``` is used to store the previous index in the source code.
```

Comment:
you can just use one backtick instead of 3 yknow

Reply:
oh I didn't knew that

Reply:
Is everything fine now?

Reply:
i don't see any changes

Reply:
Changed it